academy/4-my-own-chain/starport.md

```diff
@@ -191,6 +191,31 @@ You can observe the endpoints of the blockchain in the output of the `starport c
 
 Starport can detect any change to the source code. When it does, it immediately rebuilds the binaries before restarting the blockchain and keeping the state.
 
+## Interact via the CLI
+
+In fact, you can already interact with your running chain. With the chain running in its own shell, try in another:
```

Comment:
Try in another shell? I would prefer to be more specific here.

Reply:
Fixed.